blazor/scheduler/resources.md

```diff
@@ -587,6 +592,8 @@ The following code example depicts how to group the multiple resources on Timeli
     }
 }
 ```
+The following image depicts how the multiple resources will be displayed in Timeline Resource view accordingly.
```

Comment:
The following image shows the multiple resources rendering on the Timeline view Scheduler.

Reply:
The given line was added in place of previous one.